easypy/properties.py

```diff
@@ -41,3 +41,7 @@ def callable(*args, **kwargs):
         return property(callable, fset, fdel, doc)
     else:
         return property(fget, fset, fdel, doc)
+
+
+def add_property(obj, name, function):
+    setattr(obj.__class__, name, property(function))
```

Comment:
you can't have such a function, that takes a _specific_ object instance and sets to its not-so-specific _class_ a _specific_ function...